styleguide/source/_patterns/01-molecules/10-search-results/00-search-result/search-results-columns-external.twig

```diff
@@ -1,19 +1,13 @@
-<a href="{{ url }}"  class="layout layout_two_up-large">
-  <div class="layout_two_up-large-primary search-result">
-    <div class="search-result_columns">
-      {% include 'atoms-rule-horizontal' %}
-      <div class="search-result_label-wrapper">
-        {% include 'atoms-title-label' with { 'class' : 'search-result_label', 'content' : 'AMA Wire' } %}
-        {% include 'atoms-arrow-external' %}
-      </div>
-      
-      {% include 'atoms-h4' with { 'class' : 'search-result_columns-title', 'content' : 'Visit AMA Wire to view news and more information related to this topic.' } %}
-
-      <div class="search-result_members-only-label-wrapper">
-        {% include 'atoms-paragraph' with { 'class' : 'search-result_members-only-label', 'content' : 'AMA Members Only' } %}
-        {% include 'atoms-information' %}
-      </div>
-    </div>
+<a href="{{ url }}" class="search-results_columns-inner">
```

Comment:
.search-results_columns-inner won't work here because the layouts will be swapped. For layout-specific styles please use some sort of inheritance based on the prior comment.